### ISSUE TITLE
Y-axis location based on number of digits

### ISSUE THREAD

**Issue by charlespax** [PaxInstruments/t400-firmware#221]
When the temperature has two digits the graph y-axis looks like this.
![img_20160713_091111](https://cloud.githubusercontent.com/assets/281642/16788785/76d805b4-48da-11e6-9969-df062ab8bc67.jpg)

In commit 43543b7205458c8e6638093725c884eada44a73c the y-axis will shift to the right when the number of digits increases from two to three like this.
![img_20160713_091122](https://cloud.githubusercontent.com/assets/281642/16788819/b83ba8ee-48da-11e6-8c90-77b54c40b0db.jpg)

In the most recent commit 9fe18bb790af0158d80564159aebaab4549c275b the y-axis does not shift to the right. It looks like this.
![img_20160713_090044](https://cloud.githubusercontent.com/assets/281642/16788839/dd37aec2-48da-11e6-97e3-5dd9e4b47184.jpg)

This is probably happening around line 155 in functions.cpp
```
   // Calculate the number of axes digits to display
 -  if(graphMin + graphScale*4 > 999 || graphMin < -99) {		 +  axisDigits = 2;
 +  if((min + (graphScale*4)) > 9999 || min < -999) {
      axisDigits = 4;		      axisDigits = 4;
    }		    }
 -  else if(graphMin + graphScale*4 > 99 || graphMin < -9) {		 +  else if((min + (graphScale*4)) > 999 || min < -99) {
      axisDigits = 3;		      axisDigits = 3;
    }		    }
 -  else {		 +
 -    axisDigits = 2;		 +  return;
```

**Comment by charlespax** [comment]
Additional information:

If we start with two digit temperatures like this...
![img_20160713_093458](https://cloud.githubusercontent.com/assets/281642/16789143/9366d248-48dd-11e6-811d-42105a029423.jpg)
and heat things up, we get wonky graphics.
![img_20160713_093503](https://cloud.githubusercontent.com/assets/281642/16789148/a5ee657a-48dd-11e6-9dc9-773526295c55.jpg)

But if we heat the thermocouple before turning on the T400, we can start with only three digit numbers. Then we get a nice graph like this.
![img_20160713_093525](https://cloud.githubusercontent.com/assets/281642/16789171/d33fa174-48dd-11e6-8ce0-1b0343d768df.jpg)
We can keep our nice graph even if we let things cool down.
![img_20160713_093534](https://cloud.githubusercontent.com/assets/281642/16789175/e535f18a-48dd-11e6-9f7d-b620d554d1a2.jpg)

Also notice that in the original firmware the two digits are right justified while on the newer firmware they are left justified. They should be right justified.
![img_20160713_093534 copy](https://cloud.githubusercontent.com/assets/281642/16789267/a9691e06-48de-11e6-9d6f-70caca3fd7e3.jpg)

**Comment by charlespax** [comment]
Also weird with negative numbers.
![img_20160713_111627](https://cloud.githubusercontent.com/assets/281642/16790818/87064f84-48eb-11e6-9bb6-b7cdb6eef3b3.jpg)

In that image you may have also noticed the [double negative](https://github.com/PaxInstruments/t400-firmware/issues/222).

**Comment by protological** [comment]
The logic for the number of character spaces needs to be updated for the new 1/10th temperature values.  I have some updated code and will push a fixed branch

**Comment by protological** [comment]
Fixed in commit 78bd24936d339fcaf88add8d64a5a1af9bf23ccc
